src/overseer/api.clj

```diff
@@ -21,14 +21,16 @@
               [executor :as exc]
               [worker :as worker])
             [overseer.store.datomic :as store.datomic]
+            [overseer.store.jdbc :as store.jdbc]
             [loom.graph :as loom]))
 
 (defn store
   "Return a Store implementation based on the store type and type-specific
   configuration in `config` map"
   [config]
   (case (config/store-type config)
-    :datomic (store.datomic/store (:uri (config/datomic-config config)))))
+    :datomic (store.datomic/store (:uri (config/datomic-config config)))
+    :jdbc (store.jdbc/store (config/jdbc-config config))))
```

Comment:
The structure is (now) uniform and there's not really any specific validations or fields examined at all actually, JDBC options are just passed in unchanged, for example. I'm fine with this - breaking out would make more sense as structure diverges or validations get more sophisticated